Automattic/wp-calypso
# Nav Unification: Update image assets and CSS classes for new positioning

#### Changes proposed in this Pull Request

* Update image assets
* Update modal copy

#### Testing instructions



* Visit generated live Calypso link below
* Remove preference `nav-unification-modal` using the debug badge in the bottom right
* Observe new image assets for both mobile and desktop, should reflect below (in order)

<table>
<tr>
<td>Desktop</td><td>Mobile</td>
</tr>
<tr>
<td><img width="1440" alt="Desktop - Modal 4" src="https://user-images.githubusercontent.com/67021691/108780267-28e96400-7536-11eb-809a-d07a6b9277cf.png"></td>
<td><img width="375" alt="Mobile - Modal 7" src="https://user-images.githubusercontent.com/67021691/108780296-343c8f80-7536-11eb-94bf-d5258130a9f3.png"></td>
</tr>
<tr>
<td><img width="1440" alt="Desktop - Modal 5" src="https://user-images.githubusercontent.com/67021691/108780340-47e7f600-7536-11eb-9b8c-5b98f3da5591.png"></td>
<td><img width="375" alt="Mobile - Modal 8" src="https://user-images.githubusercontent.com/67021691/108780343-4a4a5000-7536-11eb-91b3-7bb60de4e488.png"></td>
</tr>
<tr>
<td><img width="1440" alt="Desktop - Modal 6" src="https://user-images.githubusercontent.com/67021691/108780385-59c99900-7536-11eb-8cbf-102cfb0a0e70.png"></td>
<td><img width="375" alt="Mobile - Modal 9" src="https://user-images.githubusercontent.com/67021691/108780394-5cc48980-7536-11eb-9d6a-63a14eb28cbf.png">
</td>
</tr>
</table>




Fixes https://github.com/Automattic/wp-calypso/issues/50038
Fixes https://github.com/Automattic/wp-calypso/issues/50322
